README.md

```diff
@@ -794,6 +794,24 @@
 
     ```
 
+  - Remove all trailing whitespace
+
+    ```javascript
+    // bad
+    function() {∙∙
+      var name;
+    ∙∙
+      // ... stuff ...
+    }
+
+    // good
+    function() {
+      var name;
+
+      // ... stuff ...
+    }
+    ```
+
```

Comment:
Sublime Text has the [`trim_trailing_white_space_on_save`](http://sublime-text-unofficial-documentation.readthedocs.org/en/latest/reference/settings.html#whitespace-and-indentation) setting for this.